src/library.rs

```diff
@@ -132,7 +134,7 @@ pub enum Fundamental {
     Unsupported,
 }
 
-pub const FUNDAMENTAL: [(&'static str, Fundamental); 31] = [
+const FUNDAMENTAL: &'static [(&'static str, Fundamental)] = &[
```

Comment:
This is necessary because [T; 33] does not implement any traits anymore (and we need Iterator below), it only goes up to 32.